documentation/book/ref-topic-operator-usage-recommendation.adoc

```diff
@@ -5,11 +5,49 @@
 [id='topic-operator-usage-recommendations-{context}']
 = Topic Operator usage recommendations
 
-* Be consistent and always operate on `KafkaTopic` resources or always operate on topics directly. Avoid routinely using both methods for a given topic.
-* When creating a `KafkaTopic` resource:
-** Remember that the name cannot be changed later.
-** Choose a name for the `KafkaTopic` resource that reflects the name of the topic it describes.
-** Ideally the `KafkaTopic.metadata.name` should be the same as its `spec.topicName`. To do this, the topic name will have to be a link:https://github.com/kubernetes/community/blob/master/contributors/design-proposals/architecture/identifiers.md[valid {KubernetesName} resource name^].
-* When creating a topic:
-** Remember that the name cannot be changed later.
-** It is best to use a name that is a link:https://github.com/kubernetes/community/blob/master/contributors/design-proposals/architecture/identifiers.md[valid {KubernetesName} resource name^], otherwise the operator will have to modify the name when creating the corresponding `KafkaTopic`.
+When working with topics, be consistent and always operate on either `KafkaTopic` resources or topics directly. Avoid routinely switching between both methods for a given topic.
+
+Use topic names that reflect the nature of the topic, and remember that names cannot be changed later.
+
+If creating a topic in Kafka, use a name that is a valid {KubernetesName} resource name, otherwise the Topic Operator will need to create the corresponding `KafkaTopic` with a different name which obeys the {KubernetesName} rules.
+
+.Kafka topic naming conventions
+
+Kafka and {KubernetesName} impose their own validation rules for the naming of topics in Kafka and `KafkaTopic.metadata.name` respectively.
+There are valid names for each which are invalid in the other.
+
+The `spec.topicName` property provides a way to create a topic in Kafka with a name that would be invalid for the `KafkaTopic` in {PlatformProductName}, link:https://github.com/kubernetes/community/blob/master/contributors/design-proposals/architecture/identifiers.md[(according to the recommendations for identifiers and names in {KubernetesName})^].
```

Comment:
"provides a way" almost sounds like a recommendation. I suggest something along the following lines:


Using the `spec.topicName` property, it is possible to create a valid topic in Kafka with a name that is invalid for the `KafkaTopic` in {PlatformProductName}.

I feel the link to the Kubernetes article should be introduced earlier.

Reply:
Better. I moved the link up too.